backends/dpdk/dpdkAsmOpt.h

```diff
@@ -290,6 +291,68 @@ class ShortenTokenLength : public Transform {
     }
 };
 
+// This pass identify redundant copies/moves and eliminate it
+// and propogate original sources.
+class CopyPropagationAndElimination : public Transform {
+    std::unordered_map<cstring, int> usesInfo;
```

Comment:
so what are all these cstrings?

Reply:
you seem to depend on the fact that names are unique. you should probably document that.

Reply:
done